x/ecocredit/basket/keeper/msg_update_curator_test.go

```diff
@@ -0,0 +1,112 @@
+package keeper
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/regen-network/gocuke"
+	"github.com/stretchr/testify/require"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	api "github.com/regen-network/regen-ledger/api/regen/ecocredit/basket/v1"
+	"github.com/regen-network/regen-ledger/types/testutil"
+	types "github.com/regen-network/regen-ledger/x/ecocredit/basket/types/v1"
+)
+
+type updateBasketCurator struct {
+	*baseSuite
+	alice sdk.AccAddress
+	bob   sdk.AccAddress
+	res   *types.MsgUpdateCuratorResponse
+	err   error
+}
+
+func TestUpdateBasketCurator(t *testing.T) {
+	gocuke.NewRunner(t, &updateBasketCurator{}).Path("./features/msg_update_curator.feature").Run()
+}
+
+func (s *updateBasketCurator) Before(t gocuke.TestingT) {
+	s.baseSuite = setupBase(t)
+	s.alice = s.addrs[0]
+	s.bob = s.addrs[1]
+}
+
+func (s *updateBasketCurator) ABasketWithPropertiesAndCuratorAlice(a gocuke.DocString) {
+	var basket types.Basket
+	err := json.Unmarshal([]byte(a.Content), &basket)
+	require.NoError(s.t, err)
+
+	err = s.k.stateStore.BasketTable().Save(s.ctx, &api.Basket{
+		BasketDenom:      basket.BasketDenom,
+		Name:             basket.Name,
+		CreditTypeAbbrev: basket.CreditTypeAbbrev,
+		Curator:          s.alice,
+	})
+	require.NoError(s.t, err)
```

Comment:
recently discovered this issue https://github.com/cosmos/cosmos-sdk/issues/12332

i dont think its really necessary to change the tests here in this PR, unless you want to, but just adding a note here that we probably want to switch to gotest.tools/v3/assert for future tests. tracking: https://github.com/regen-network/regen-ledger/issues/1487

Reply:
Oof. I do like the methods available in require but makes sense. There is a command line tool to automate the migration so we can handle all at once in a separate pull request. Also see https://github.com/regen-network/regen-ledger/issues/1487#issuecomment-1247343297